src/wormhole/session/item/mod.rs

```diff
@@ -11,6 +12,9 @@ use spawn::spawn;
 mod hatch;
 use hatch::hatch;
 
+mod rename_gotchi;
+use rename_gotchi::rename;
```

Comment:
would complain because all of them are named i.e. "hatch" and not "hatch_item", but because gotchi aren't items I can see why you named it "rename_gotchi". okay.

Reply:
I changed the name of the Ask to NicknameGotchi, this should probably be nickname_gotchi but it really doesn't matter